riscv/processor.h

```diff
@@ -168,6 +168,28 @@ struct state_t
   csr_t_p stimecmp;
   csr_t_p vstimecmp;
 
+  // Smcsrind / Sscsrind
+  sscsrind_sel_csr_t::sscsrind_sel_csr_t_p miselect;
```

Comment:
Do these all need to be here? They don't seem to be used today except in `state_t::reset()` so they could be local to that method.

Reply:
Right, will address it in the update.